my-app/src/components/Post/Post.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react'
+import { IPost } from '../PostList/PostList'
+import "./style.css"
+
+
+
+const Post = ({ title, date, image, description, id }: IPost) => {
+
+   return (
+      <li className='post__container'>
+         <div className='post__date'>{date}</div>
+         {image ? <img className='post__img' src={image} alt="img" /> : ""}
```

Comment:
We don't need to render an empty string. If image is null/undefined - then do it like this:

`{image && <img className='post__img' src={image} alt="img" />}`